src/EnergyPlus/ScheduleManager.cc

```diff
@@ -231,7 +231,7 @@ namespace ScheduleManager {
         Array1D_bool AllDays(MaxDayTypes);
         Array1D_bool TheseDays(MaxDayTypes);
         bool ErrorHere;
-        int SchNum;
+        int SchNum{};
```

Comment:
I'm curious why this is necessary if you're not actually initializing to a value?

Reply:
It would automatically initialize it to 0 if I used the `{}`. I will change it to `{0}` for clarity 👍🏽

Reply:
I see. Thanks for clarifying.